src/main/java/org/openstreetmap/atlas/geography/atlas/items/AtlasEntity.java

```diff
@@ -194,6 +196,32 @@ public String toDiffViewFriendlyString()
      */
     public abstract LocationIterableProperties toGeoJsonBuildingBlock();
 
+    /**
+     * A base method that creates a GeoJSON feature with properties from the tags. Note that it does
```

Comment:
So that sub-classing this is fool-proof, I would go the following way instead (names non-binding):
1. Create an abstract method called `asGeojsonFeature` that sub-classes will have to implement to compile
1. Have this `asGeoJsonFeature` method `protected`, and renamed to `asGeoJsonFeatureWithoutGeometry`, so it is only used by the sub classes and not used by customers by mistake.

Reply:
Good call, done.